### DISCUSSION TITLE
Is there a way to use two different forms on the feedback widget ?

### DISCUSSION THREAD

**Discussion by Marlony-Gabriel** [squidfunk/mkdocs-material#5462] [Q&A]
Hello again.
I'm improving my site documentation, and I'm now using google forms to take opinions from users.
That part is okay. 
Now, i'm put some vídeos and I wish to get separeted opinions from pages with and withou videos.
What I'm trying is using the feedback widget for this but for this I'll need to provide differet links according to the pages where the user are providing feeback. Something like:

```
feedback:
      title: Esta página foi útil ?
      ratings:
        - icon: material/thumb-up-outline
          name: Esta página me ajudou.
          data: 1
          note: >-
            Obrigado pelo Feedback!
            Ajude-nos a evoluir, respondendo nossa <a href="https://docs.google.com/forms/d/e/1FAIpQLSej6uRqfVMQc32-X8-JeTDkbLy_wm2OQtrT3VTa3WZjw-14ZQ/viewform?usp=pp_url&entry.925137426={title} - {url}" target=_blank rel="noopener">Pesquisa Rápida.</a>
        - icon: material/thumb-down-outline
          name: Esta página pode melhorar
          data: 0
          note: >- 
            Poxa, pena que não conseguimos te atender.
            Ajude-nos a melhorar respondendo nossa  <a href="https://docs.google.com/forms/d/e/1FAIpQLSej6uRqfVMQc32-X8-JeTDkbLy_wm2OQtrT3VTa3WZjw-14ZQ/viewform?usp=pp_url&entry.925137426={title} - {url}" target=_blank rel="noopener">Pesquisa Rápida.</a> 
```
If the page has a vídeo, I want to put a different link in the notes. Is that possible ?
And, if i'm nos asking too much, there is a way or any config that i can make to know how much "Clicks" the feedback form link had 
on the google analytics ?

Thank you again for the great work.

**Comment by squidfunk** [comment]
That's not possible, and I'd regard it as an edge case that's likely not happening for many users. You could probably achieve something like this via [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme), copying the feedback widget partial / logic and use a front matter property to switch the partials. That will definitely work, but it'll be some hoops to jump through.
